### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by stephpy** [spf13/spf13-vim#71]
Hi, there is an error during the installation, the file

```
source $HOME/.vim/bundle/snipmate-snippets/snippets/support_functions.vim
```

is missing, when you launch

```
vim +BundleInstall! +BundleClean +q
```

By this way, i have a lot of bundles which are not installed, i have to relaunch the boostrap.sh to install them. 

Is it a problem to test if this file exists to fix this error ? I can make a PR if you want

**Comment by stephpy** [comment]
Something else, Is there any reason to make a key mapping for "BufExplorer" but this one is not installed via vundle ?

**Comment by stephpy** [comment]
Fixed on pr #72
